lib/src/widget/menu/app_menu.dart

```diff
@@ -0,0 +1,152 @@
+import 'package:flutter/material.dart';
+import 'package:xayn_design/src/linden/linden.dart';
+import 'package:xayn_design/src/widget/unter_den_linden/unter_den_linden.dart';
+
+const _kDividerThickness = 1.0;
+
+class AppMenu extends StatelessWidget {
+  const AppMenu({
+    Key? key,
+    required this.children,
+    this.errorStr,
+    this.physics = const NeverScrollableScrollPhysics(),
+    this.onClose,
+    this.isSticky = false,
+    this.permitRoutePop = true,
+    this.start,
+    this.top,
+    this.end,
+    this.bottom,
+    this.width,
+    this.height,
+    this.textDirection = TextDirection.ltr,
+  })  : assert(
+          start == null || end == null || width == null,
+          'AppMenu needs to have horizontal constraints: start or end or width',
+        ),
+        assert(
+          top == null || bottom == null || height == null,
+          'AppMenu needs to have vertical constraints: top or bottom or height',
+        ),
+        super(key: key);
+
+  final List<Widget> children;
+  final String? errorStr;
```

Comment:
What is `errorStr` used for?

Reply:
menus [display error in a specific way](https://user-images.githubusercontent.com/12827629/154662894-18a112e3-31d0-43a5-88a2-de504e2e14a0.png) according to the design team.
I'll add a comment there.

Reply:
Maybe rename it to `errorMessage`? `errorStr` sounds a bit cryptic.

Reply:
alright 👍